### PR TITLE
update iRO Re:Start files

### PR DESCRIPTION
**Pull request by allanon**
- [ ] QA Review

### CODE REVIEW

**Comment by alisonrag** [comment]
@allanon 
Will you update this pull with table files?

**Comment by zPhotons** [comment]
thank you

**Comment by zPhotons** [comment]
Allanon quick question how can you extract those packets? I am eager to learn would you mind messaging me? Thank you.

File: tables/iRO/Renewal/shuffle.txt [hunk 1]
```diff
@@ -1,56 +1,216 @@
...
+0438 0955
+035F 091F
+0360 086E
+0361 0934
+0362 0875
+0363 095B
+0364 086A
+0365 086F
+0366 091B
+0367 089E
```
Comment (c4c1n6kr3m1) [inline tables/iRO/Renewal/shuffle.txt]: this

File: tables/iRO/Renewal/shuffle.txt [hunk 2]
```diff
@@ -1,56 +1,216 @@
...
+0802 0936
+07EC 0937
+07E4 022D
+0436 0891
+02C4 0838
+0281 087A
+0202 08A0
+022D 0966
+023B 0140
+0367 0873
```
Comment (c4c1n6kr3m1) [inline tables/iRO/Renewal/shuffle.txt]: and this???096A???

**Pull request merged**